cmd/util/track_params.go

```diff
@@ -0,0 +1,70 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package cmdutil
+
+import (
+	"time"
+
+	"github.com/elastic/cloud-sdk-go/pkg/plan"
+	"github.com/elastic/cloud-sdk-go/pkg/plan/planutil"
+
+	"github.com/elastic/ecctl/pkg/ecctl"
+)
+
+// DefaultTrackFrequencyConfig provides sane defaults for Plan change tracking.
+var DefaultTrackFrequencyConfig = plan.TrackFrequencyConfig{
+	PollFrequency: time.Second * 5,
+	MaxRetries:    3,
+}
+
+// TrackParamsConfig is used to create TrackParams which print the output / track.
+type TrackParamsConfig struct {
```

Comment:
Should this struct have a validation method?

Reply:
@marclop should it? 😄

Reply:
Added some testing and defaults for the App field.